features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/Workflow/ApplicationCreation.jsx

```diff
@@ -376,9 +376,9 @@ function ListLabels() {
                         <Typography variant='body2' color='textSecondary' component='p'>
                             <FormattedMessage
                                 id='Workflow.ApplicationCreation.List.empty.content.applicationcreations'
-                                defaultMessage={'There are no workflow pending requests for application creation.'
-                                    + 'It is possible to approve or reject workflow pending requests of application'
-                                    + 'creation. Workflow Approval Executor needs to be enabled to approve or '
+                                defaultMessage={'There are no pending workflow requests for application creation.'
+                                    + 'You can approve or reject pending workflow requests for application '
```

Comment:
I don't think we need to say "You can approve or reject pending workflow requests for application" after saying there are no pending tasks. Isn't it confusing? @kavishkafernando wdyt?

Reply:
What if it's a user coming to that page for the first time?

Reply:
what if we say there are no pending tasks to be approved/denied?  and I also think we don't need to mention about "Workflow Approval Executor" here. it should be in docs only.

Reply:
IMO the last sentence should be in the UI as an error message or a note.

Reply:
According to the other UIs that we have in the Admin Portal we usually give a small description of the feature if there is no data to be shown. Shall we provide a doc link in the description on how to configure workflows? @bhathiya WDYT?

The issue is we cant identify whether the user is coming for the first time to this UI or whether he is a user already using this feature who has approved previous requests.

Reply:
@ruthryi @kavishkafernando what is our conclusion here?

Reply:
We discussed offline and IMO we shouldn't talk about server-level configs in UI. It doesn't make sense for tenants, for example in the cloud. If those configs are in the UI and the user who sees it, has access, then we can mention. So after discussing we concluded to keep only the 1st line. @ruthryi can you please fix and merge this asap?